csrc/python_frontend/python_bindings.cpp

```diff
@@ -713,10 +713,10 @@ void initNvFuserPythonBindings(PyObject* module) {
       .def(
           "define_tensor",
           [](FusionDefinition& self,
-             std::vector<int64_t>& shape,
-             std::vector<std::optional<bool>>& contiguity,
-             PrimDataType dtype = DataType::Float,
-             bool is_cpu = false,
+             const std::vector<int64_t>& shape,
+             std::vector<std::optional<bool>> contiguity = {},
+             const PrimDataType dtype = DataType::Float,
```

Comment:
nitpick: I guess this is not an exposed c++ api so it's likely no big deal, but const-qualified value argument sounds like a bad practice. Feel free to discard my personal style~~

Reply:
> const-qualified value argument sounds like a bad practice

You are right about that for function **declarations**. For function **definitions**, const-qualified value arguments are fine: https://abseil.io/tips/109. 

However, this case is a bit tricky because it's both a declaration and a definition, so 🤷